src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-680.mdx

```diff
@@ -0,0 +1,19 @@
+---
+subject: Android agent
+releaseDate: '2022-07-27'
+version: 6.8.0
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_6.8.0.zip'
+---
+
+## New in this release
+* Add support for handling JavaScript Errors from React Native apps
+* Add a new session attribute 'RootedDevice' to signify whether the hosting device is rooted
```

Comment:
One last change: "Add a new session attribute _when native reporting is enabled_..."

Reply:
okay, will do